src/seedu/addressbook/commands/IncorrectCommand.java

```diff
@@ -16,5 +16,8 @@ public IncorrectCommand(String feedbackToUser){
     public CommandResult execute() {
         return new CommandResult(feedbackToUser);
     }
+    
+    @Override
+    public boolean isMutating() { return false; }
```

Comment:
Should avoid single line methods as it reduce the readability, cohesive with the rest of the code and may potentially introduce things like
>public void doSomething() { a; b; c; }

which is bad coding practice as the example above has multiple statements in single line.